planner/core/physical_plan_test.go

```diff
@@ -35,100 +36,62 @@ import (
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/sessionctx/stmtctx"
 	"github.com/pingcap/tidb/sessionctx/variable"
+	"github.com/pingcap/tidb/testkit"
+	"github.com/pingcap/tidb/testkit/testdata"
 	"github.com/pingcap/tidb/util/hint"
-	"github.com/pingcap/tidb/util/testkit"
+	oldtestkit "github.com/pingcap/tidb/util/testkit"
 	"github.com/pingcap/tidb/util/testleak"
-	"github.com/pingcap/tidb/util/testutil"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = Suite(&testPlanSuite{})
-var _ = SerialSuites(&testPlanSerialSuite{})
-
-type testPlanSuiteBase struct {
-	*parser.Parser
-	is infoschema.InfoSchema
-}
-
-func (s *testPlanSuiteBase) SetUpSuite(c *C) {
-	s.is = infoschema.MockInfoSchema([]*model.TableInfo{core.MockSignedTable(), core.MockUnsignedTable()})
-	s.Parser = parser.New()
-	s.Parser.SetParserConfig(parser.ParserConfig{EnableWindowFunction: true, EnableStrictDoubleTypeCheck: true})
-}
-
-type testPlanSerialSuite struct {
-	testPlanSuiteBase
-}
-
-type testPlanSuite struct {
-	testPlanSuiteBase
-
-	testData testutil.TestData
-}
-
-func (s *testPlanSuite) SetUpSuite(c *C) {
-	s.testPlanSuiteBase.SetUpSuite(c)
-
-	var err error
-	s.testData, err = testutil.LoadTestSuiteData("testdata", "plan_suite")
-	c.Assert(err, IsNil)
-}
-
-func (s *testPlanSuite) TearDownSuite(c *C) {
-	c.Assert(s.testData.GenerateOutputIfNeeded(), IsNil)
+func utilitiesSetup(tk *testkit.TestKit) (*parser.Parser, infoschema.InfoSchema, session.Session) {
+	p := parser.New()
+	p.SetParserConfig(parser.ParserConfig{EnableWindowFunction: true, EnableStrictDoubleTypeCheck: true})
+	is := infoschema.MockInfoSchema([]*model.TableInfo{core.MockSignedTable(), core.MockUnsignedTable()})
+	se := tk.Session()
+	return p, is, se
 }
 
-func (s *testPlanSuite) TestDAGPlanBuilderSimpleCase(c *C) {
-	defer testleak.AfterTest(c)()
-	store, dom, err := newStoreWithBootstrap()
-	c.Assert(err, IsNil)
-	defer func() {
-		dom.Close()
-		store.Close()
-	}()
-	se, err := session.CreateSession4Test(store)
-	c.Assert(err, IsNil)
-	_, err = se.Execute(context.Background(), "use test")
-	c.Assert(err, IsNil)
-	_, err = se.Execute(context.Background(), "set tidb_opt_limit_push_down_threshold=0")
-	c.Assert(err, IsNil)
+func TestDAGPlanBuilderSimpleCase(t *testing.T) {
+	defer testleak.AfterTestT(t)()
```

Comment:
goroutine leaks are already covered by `main_test.go`.

Please refer to other migrated tests under `planner/core` package for examples.

Reply:
e.g. https://github.com/pingcap/tidb/pull/32510